tutor/src/components/add-edit-question/index.js

```diff
@@ -10,7 +10,9 @@ import {
 
 const AddEditQuestionModals = observer(({ ux }) => {
   if(!ux.didUserAgreeTermsOfUse) {
-    return <AddEditQuestionTermsOfUse ux={ux} />;
+    return <AddEditQuestionTermsOfUse
+      onClose={() => ux.onDisplayModal(false)}
+      show={!ux.didUserAgreeTermsOfUse} />;
   }
```

Comment:
show seems like it'll always be true here since we're not rendering `AddEditQuestionTermsOfUse` at all unless it's true.

Reply:
technically it has also an if statement above so it won't render this component at all if `!ux.didUserAgreeTermsOfUse` is `false`.

Reply:
so it can be `show={true}`